doc/source/reference/arrays.nditer.rst

```diff
@@ -78,6 +78,8 @@ order='C' for C order and order='F' for Fortran order.
     ...
     0 3 1 4 2 5
 
+.. _nditer-context-manager:
+
 Modifying Array Values
 ----------------------
 
```

Comment:
The next few paragraph below this point could use some editing. It uses the term "view" in a possbily misleading way, since the writeback arrays are not actually views, they are temp buffers.

Here's my try going through it:
```
 By default, the :class:`nditer` treats the input operands as a read-only
 object. To be able to modify the array elements, you must specify either
 read-write or write-only mode using the `'readwrite'` or `'writeonly'`
 per-operand flags. 

The nditer will then yield writeable buffer arrays which you may modify. However,
because  the nditer must copy this buffer data back to the original array once
 iteration is finished, you must signal when the iteration is ended, by one of two 
methods. You may either:

 - used the nditer as a context manager using the `with` statement, and 
   the temporary data will be written back when the context is exited.
 - call to the iterator's `close` method to ensure the modified data
   is written back to the original array.

The iterator can no longer be used once either `close` is called or the its
context is exited.

When assigning to the buffers returned by the nditer, also remember that you must 
perform the assignment using the ellipsis index, as in `x[...] = `, rather than 
simply reassigning the variable name as in `x = `.
```